release/src/ShopwarePluginRelease.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Release;
+
+use Bramus\Monolog\Formatter\ColoredLineFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+
+class ShopwarePluginRelease
+{
+    /** @var LoggerInterface */
+    private $logger;
+
+    /** @var ReleaseBundler */
+    private $releaseBundler;
+
+    /** @var ReleaseTagger */
+    private $releaseTagger;
+
+    public function __construct(
+        LoggerInterface $logger,
+        ?ReleaseBundler $releaseBundler = null,
+        ?ReleaseTagger $releaseTagger = null
+    ) {
+        $this->logger = $logger;
```

Comment:
Use `$this->buildLogger()` here and just pass the name as string.

Reply:
Both files `release` and `build-release` still utilize the logger. This means if we want to add more logging in the future or add more functionality, which may be outside of `ShopwarePluginRelease`, we would have no way to access the logger.